docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -2008,3 +2008,206 @@ generic object: ALAddress
 template: x.has_no_address_explanation_help
 content: |
   Example: "I normally sleep at 5th and Main."
+---
+################## Name change questions ######################
+---
+id: What is your new name?
+question: |
+  What name do you want to be called?
+fields:
+  - First name: users[0].preferred_name.first
```

Comment:
The only issue I have here is the repeated use of the thing that we already have; the `name_fields()`. I'm having trouble thinking of better ways to organize it, but I think it's worth considering if we should refactor and move `name_fields()` to the name object instead of the person object. That's pretty hard though, and I am afraid it would break a lot of things.

Reply:
Could just keep the current behavior, deprecate it, and add `name_fields()` to the name object if we want a longer transition time or to grandfather the old behavior in while establishing a new convention.